test/e2e/reboot.go

```diff
@@ -160,13 +168,24 @@ func rebootNode(c *client.Client, provider, name string, result chan bool) {
 		result <- false
 		return
 	}
-	Logf("Calling reboot on %s", name)
-	rebootCmd := "sudo reboot"
+	var rebootCmd string
```

Comment:
I'd suggest factoring the "Reboot the node" part out into a separate utility function (sorry - github will only allow me to comment on changed lines, otherwise I would have attached this comment to line 157)